content/docs/2.6/scalers/datadog.md

```diff
@@ -99,3 +107,25 @@ spec:
     authenticationRef:
       name: keda-trigger-auth-datadog-secret
 ```
+
+## Polling intervals and Datadog rate limiting
+
+[API Datadog endpoints are rate
+limited](https://docs.datadoghq.com/api/latest/rate-limits/). Depending on the
+state of the `ScaledObject` there are two different parameters to control how
+often (per `ScaledObject`) we query Datadog for a metric.
+
+When scaling from 0 to 1, the polling interval is controlled by KEDA, using [the
+`spec.pollingInterval` parameter in the `ScaledObject`
+definition](../concepts/scaling-deployments.md#pollinginterval). For example, if
```

Comment:
Drop the `.md` suffix from the link reference.

Reply:
done